src/Xamarin.Android.Build.Tasks/Xamarin.Android.Tooling.targets

```diff
@@ -41,16 +41,27 @@ projects.
     </ItemGroup>
   </Target>
 
-  <Target Name="_ResolveSdks" DependsOnTargets="_GetReferenceAssemblyPaths">
+  <PropertyGroup Condition=" '$(UsingAndroidNETSdk)' == 'True' ">
+    <_ResolveSdksDependsOnTargets>ResolveFrameworkReferences</_ResolveSdksDependsOnTargets>
+    <TargetFrameworkDirectory>$(MSBuildThisFileDirectory)..\lib\MonoAndroid\v10.0\</TargetFrameworkDirectory>
```

Comment:
Just as a note, this path will be changing shortly in an upcoming PR when these assemblies are removed from the SDK.